pom.xml

```diff
@@ -99,12 +100,32 @@
 			</dependency>
 		</dependencies>
 	</dependencyManagement>
-	<!--TODO: Move transport related dependencies here pending https://jira.spring.io/browse/XD-3337  -->
 	<dependencies>
 		<dependency>
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-stream</artifactId>
 		</dependency>
+		<!-- Cloud connector dependencies -->
+		<!-- Lattice connector dependency to create services info from lattice -->
+		<dependency>
```

Comment:
I think they don't necessarily need to be transitive if the module itself is added as a dependency to other project. and, I believe these cloud dependencies would be left with the developer in that case. what do you think?